safe/impact_function/test/test_impact_function.py

```diff
@@ -384,9 +386,12 @@ def test_scenarii(self):
             'raster_continuous_on_raster_population': False,
         }
 
+        # If we want to invert the selection.
+        invert = False
+
         path = standard_data_path('scenario')
         for scenario, enabled in scenarii.iteritems():
```

Comment:
BTW I think scenarios is a more established plural for scenario that scenarii ....